src/lasp_core.erl

```diff
@@ -912,6 +912,34 @@ write(Type, Value, Key, Store) ->
 increment_counter(Counter) ->
     Counter + 1.
 
+%% @private
+store_delta(Type, Counter, Delta, DeltaMap0) ->
+    MaxDeltaSlots = application:get_env(lasp, delta_mode_max_slots, 10),
```

Comment:
This `gen_env` is extremely expensive.  Can you use mochiglobal like I did for the `delta_mode` setting?